src/index.html

```diff
@@ -12,7 +12,48 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        class="product-card__image"
+        alt="imac"
+      />
+
+      <div class="product-card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
```

Comment:
Avoid using <br> to create a new line for text content. Consider wrapping the text into separate elements or using CSS to manage the layout.